File: Assignment/Module-1(fundamentals/p6.cpp
#include<stdio.h>
main()
{
	float principal,rate,time,simpleinterest;
	printf("Enter principal amount:");
	scanf("%f",&principal);
	printf("Enter rate of interest:");
	scanf("%f",&rate);
	printf("Enter time (in years):");
	scanf("%f",&time);
	simpleinterest=(principal*rate*time)/100;
	printf("Simple Interest = %.2f\n",simpleinterest);
}

File: Assignment/flutter dart assignment module-1/p6.dart
import 'dart:io';

void main() {
  double principal, rate, time;

  stdout.write("Enter the principal amount: ");
  principal = double.parse(stdin.readLineSync()!);

  stdout.write("Enter the annual interest rate (as a percentage): ");
  rate = double.parse(stdin.readLineSync()!);

  stdout.write("Enter the time (in years): ");
  time = double.parse(stdin.readLineSync()!);

  double simpleInterest = (principal * rate * time) / 100;

  print("Simple Interest: $simpleInterest");
}
